config/jobs/kubernetes/sig-node/containerd.yaml

```diff
@@ -448,6 +448,11 @@ periodics:
       - --
       - --check-leaked-resources
       - --extract=ci/latest
+      # Note: The GCE Node image used may have a dependency on the nvidia-driver-installer image defined in https://github.com/kubernetes/kubernetes/blob/master/test/e2e/testing-manifests/scheduling/nvidia-driver-installer.yaml
+      # If updating the image defined here, the cos-gpu-installer image may need to updated to support the corresponding COS image.
+      - --env=KUBE_NODE_OS_DISTRIBUTION=gci
+      - --env=KUBE_GCE_NODE_IMAGE=cos-97-16919-189-5
```

Comment:
Why do we have to pin to cos-97? 

Cant we also update the `cos-gpu-installer` image to `v2.0.29` and use the corresponding cos image?

Reply:
I'm ok with stabilising with the image we point to already then updating things later - it can be really tricky to bump that image :(